src/Client.js

```diff
@@ -155,66 +165,76 @@ export default class Client extends EventTarget {
    * Get all the messages for the passed in alias
    *
    * Note the currently logged in account must own the alias associated with the aliasName.
-   * @param {string} aliasName the name of the alias to get messages for.
-   * @param {string[]} [interlocutors] an optional list of the senders and recipients of the messages.
-   * @param {Date} [sinceTime] an optional date to limit the request by. only receive messages since this date.
+   * @param {Object} options
+   * @param {string} options.aliasName the name of the alias to get messages for.
+   * @param {string[]} [options.interlocutors] an optional list of the senders and recipients of the messages.
+   * @param {Date} [options.sinceTime] an optional date to limit the request by. only receive messages since this date.
    * @returns {Promise<Message[]>} a list of messages which pass the filters.
    */
-  getMessagesForAlias(aliasName, interlocutors, sinceTime) {
+  getMessages({ aliasName, interlocutors, sinceTime } = {}) {
+    aliasName = aliasName ?? this.account.username;
     return this.messages.getMessagesForAlias(aliasName, interlocutors, sinceTime);
   }
 
   /**
    * Get a message which has the passed in messageId, and was sent or received by the passed in aliasName.
    *
    * Note the currently logged in account must own the alias associated with the aliasName.
-   * @param {string} aliasName the name of the alias which sent or received the message.
-   * @param {string} messageId the id of the message to get.
+   * @param {Object} options
+   * @param {string} options.aliasName the name of the alias which sent or received the message.
+   * @param {string} options.messageId the id of the message to get.
    * @returns {Promise<Message>} The model of the message with the associated id.
    */
-  getMessageById(aliasName, messageId) {
+  getMessageById({ aliasName, messageId } = {}) {
+    aliasName = aliasName ?? this.account.username;
     return this.messages.getMessage(aliasName, messageId);
```

Comment:
Can there be messages with the same id and different aliases? If not, why do we need to provide the alias to get the message?

Reply:
So the idea of aliases was that so a user could have a separation of "workspace". The same account could have multiple aliases, one for work, one for friends, etc. We check this separation on the server. In the getMessageById endpoint, removing the alias isn't any work at all but may cause a problem e.g., when the user gets notified of the ID of a message that is sent to their work alias, but before they request it, they change their alias to the alias they use for friends. Sending the current alias along with the ID ensures they won't fetch a message that was not intended for their current alias.

Reply:
If we have a notion of "current" alias, we can check if their current alias has access to that message and not return it if not intended for the current alias. Would that make sense?

Reply:
The current alias is not maintained in the server. Only the client knows what is the current alias.

Reply:
Yes, this can be a client-side abstraction